auditlog/admin.py

```diff
@@ -34,3 +41,7 @@ def has_change_permission(self, request, obj=None):
 
     def has_delete_permission(self, request, obj=None):
         return False
+
+    def get_queryset(self, request):
```

Comment:
Why do we need this here?

Reply:
Because we need to access the request in order to append the cid to it for filtering